### ISSUE TITLE
Is there any option to close Simulator after Appium Execution.

### ISSUE THREAD

**Issue by aruvimal** [appium/appium#12949]
## The problem

**Is there any option to close Simulator after Appium Execution. It is tendious process to close the simulator after the exeuction. there are many simulators are remains open and very difficult to close it manually**

## Code
public class appiumiOSTest 	
{
		
		@Test
		public void manin1 () throws MalformedURLException
		{	
			
			DesiredCapabilities caps = new DesiredCapabilities();
			caps.setCapability("automationName", "XCUITest");
			caps.setCapability("noReset", "false");
			caps.setCapability("fullReset", "true");
			caps.setCapability("platformName", "iOS");
			caps.setCapability("platformVersion", "12.1");
			caps.setCapability("deviceName", "iPhone X");
			caps.setCapability(CapabilityType.BROWSER_NAME,"safari");
                         URL url = new URL("http://127.0.0.1:4723/wd/hub");
			
			WebDriver driver = new AppiumDriver<MobileElement> (url,caps);
			driver.manage().timeouts().implicitlyWait(Constants.DEVICE_WAIT, TimeUnit.SECONDS);
			driver.manage().timeouts().setScriptTimeout(Constants.SYNC_WAIT, TimeUnit.SECONDS);
			driver.manage().timeouts().pageLoadTimeout(Constants.WAIT, TimeUnit.SECONDS);
			
			driver.get("http://www.google.com");
			driver.quit();
}
}

**Comment by KazuCocoa** [comment]
What about `resetOnSessionStartOnly` https://github.com/appium/appium-xcuitest-driver ?
Asking in https://discuss.appium.io/ also helps.

**Comment by aruvimal** [comment]
Yeah it is closing the simulator but session retain for the next time execution. i want simulator to open every time with apple logo.

**Comment by KazuCocoa** [comment]
How about `enforceFreshSimulatorCreation` https://github.com/appium/appium-xcuitest-driver/pull/859
Or you can create/destroy simulator every time if your machine has no iOS 12.1 x iPhone X instance. (Appium creates it)

**Comment by mykola-mokhnach** [comment]
There is also `shutdownOtherSimulators` cap, which might be in use (although it requires the corresponding feature to be enabled on the server side).
